Jenkinsfile

```diff
@@ -17,6 +17,17 @@ pipeline {
         junit 'features/reports/*.xml'
       }
     }
+  }
 
+  post {
+    always {
+      sh 'docker run -i --rm -v $PWD:/src -w /src alpine/git clean -fxd'
```

Comment:
Is this to cleanup root-owned files? Looks like a hack (neat, I admit :) ), I wonder if it would be much work to actually not create them in the first place.